File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1031/BusinessApplication/BA.Web.10.0/Models/Shared/User.shared.cs
namespace $safeprojectname$
{
    /// <summary>
    /// Teilklasse, die den User-Typ erweitert, durch den freigegebene Eigenschaften und Methoden hinzugefügt werden.
    /// Diese Eigenschaften und Methoden sind sowohl für die Server- als auch für die Clientanwendung verfügbar.
    /// </summary>
    public partial class User
    {
        /// <summary>
        /// Gibt den Benutzeranzeigenamen zurück, welcher standardmäßig dem FriendlyName entspricht.
        /// Wenn FriendlyName nicht festgelegt ist, wird der Benutzername zurückgegeben.
        /// </summary>
        public string DisplayName
        {
            get
            {
                if (!string.IsNullOrEmpty(this.FriendlyName))
                {
                    return this.FriendlyName;
                }
                else
                {
                    return this.Name;
                }
            }
        }
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1031/BusinessApplication/BA.Web.10.0/Models/Shared/User.shared.vb
Namespace Web

    Partial Public Class User
        ''' <summary>
        ''' Gibt den Benutzeranzeigenamen zurück, welcher standardmäßig dem FriendlyName entspricht.
        ''' Wenn FriendlyName nicht festgelegt ist, wird der Benutzername zurückgegeben.
        ''' </summary>
        Public ReadOnly Property DisplayName() As String
            Get
                If Not String.IsNullOrEmpty(Me.FriendlyName) Then
                    Return Me.FriendlyName
                Else
                    Return Me.Name
                End If
            End Get
        End Property
    End Class
End Namespace